package.json

```diff
@@ -57,13 +58,12 @@
     "npm-keyword": "^4.1.0",
     "opn": "^3.0.2",
     "package-json": "^2.1.0",
-    "parse-help": "^0.1.1",
     "read-pkg-up": "^1.0.1",
     "repeating": "^2.0.0",
     "root-check": "^1.0.0",
     "sort-on": "^1.0.0",
     "string-length": "^1.0.0",
-    "tabtab": "^1.3.0",
+    "tabtab": "1.4.x",
```

Comment:
Yeah I know :/

I was't aware it was a breaking change, and really don't want to bump a major version because of a misconfigured Babel and yo integration (this `.default` thing ..)

I'll keep at that way (using `range`) and will publish tabtab@1.4 alongside this pull request.